From 58d00937b4a7df3965529b2b813c08d724b293be Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@freescale.com>
Date: Tue, 8 Aug 2017 09:46:53 +0800
Subject: [PATCH 2342/5242] MLK-16157: ARM64: dts: change the video mode for
 hdmi audio

commit  bb43bec5a3625a5b1f31adec9045d37b359e756b from
https://source.codeaurora.org/external/imx/linux-imx.git

change the video mode for hdmi audio according hdmi video device
node

Signed-off-by: Shengjiu Wang <shengjiu.wang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
index 252f10b..3d91400 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq-evk.dts
@@ -73,7 +73,7 @@
 		compatible = "fsl,imx-audio-cdnhdmi";
 		model = "imx-audio-hdmi";
 		audio-cpu = <&sai4>;
-		video-mode = <95>;	/* #16: 1920x1080p@60HZ 16:9 */
+		video-mode = <16>;	/* #16: 1920x1080p@60HZ 16:9 */
 					/* #95: 3840x2160p@30Hz 16:9 */
 	};
 
-- 
1.7.9.5

